contrib/lkt/language/parser.py

```diff
@@ -2579,6 +2579,23 @@ def called_decl():
             ),
             # If not a generic decl, fall back on the initially found decl
             default_val=called_decl
+        )._.match(
+            # Here, if we found a declaration but it's not one of the
+            # statically callable declarations (so FunDecl and TypeDecl), we
```

Comment:
I have trouble understanding this, I feel this is ambiguous: are FunDecl and TypeDecl statically callable declarations? If so I would suggest to write “i.e.” instead of “so”, and if not, write “so it’s one of …”

Reply:
This one does not look resolved: unresolving it

Reply:
I don't understand what I did, but it should be fixed now